tools/ete/ete_lineage_generator.xml

```diff
@@ -0,0 +1,147 @@
+        <conditional name="ranks">
+            <param name="levels" type="select" label="Taxonomic levels" help="Taxonomic levels to include in the output">
+                <option value="primary" selected="true">Primary</option>
+                <option value="full">Full</option>
+                <option value="manual">Manual</option>
+            </param>
+            <when value="manual">
+                <param name="manualranks" type="select" multiple="True" label="Manual selection of ranks">
+                    <option value="superkingdom">superkingdom</option>
+                    <option value="kingdom" selected="True">kingdom</option>
+                    <option value="subkingdom">subkingdom</option>
+                    <option value="superphylum">superphylum</option>
+                    <option value="phylum" selected="True">phylum</option>
+                    <option value="subphylum">subphylum</option>
+                    <option value="superclass">superclass</option>
+                    <option value="class" selected="True">class</option>
+                    <option value="subclass">subclass</option>
+                    <option value="infraclass">infraclass</option>
+                    <option value="cohort">cohort</option>
+                    <option value="superorder">superorder</option>
+                    <option value="order" selected="True">order</option>
+                    <option value="suborder">suborder</option>
+                    <option value="infraorder">infraorder</option>
+                    <option value="parvorder">parvorder</option>
+                    <option value="superfamily">superfamily</option>
+                    <option value="family" selected="True">family</option>
+                    <option value="subfamily">subfamily</option>
+                    <option value="tribe">tribe</option>
+                    <option value="subtribe">subtribe</option>
+                    <option value="genus" selected="True">genus</option>
+                    <option value="subgenus">subgenus</option>
+                    <option value="species group">species group</option>
+                    <option value="species subgroup">species subgroup</option>
+                    <option value="species" selected="True">species</option>
+                    <option value="subspecies">subspecies</option>
+                    <option value="varietas">varietas</option>
+                    <option value="forma">forma</option>
+                </param>
+            </when>
+            <when value="primary" />
+            <when value="full" />
+        </conditional>
+        <conditional name="compresscond">
+                <param name="compress" type="select" label="Fill unnamed ranks" help="Fill unnamed ranks with super/sub ranks (see -l)" />
```

Comment:
Still not right, this line should be substituted with:
```xml
            <param name="compress" type="select" label="Fill unnamed ranks" help="Fill unnamed ranks with super/sub ranks (see -l)">
                <option value="-c">Yes</option>
                <option value="">No</option>
            </param>
```

Also the indentation of this block is wrong.